app/controllers/api/v1/allocations_actions.rb

```diff
@@ -1,7 +0,0 @@
-module Api::V1
-  module AllocationsActions
-    def included_relationships
-      IncludeParamHandler.new(params).call || AllocationSerializer::SUPPORTED_RELATIONSHIPS
```

Comment:
Argh!! This is the file that caused us so much grief when testing - it defaults to returning all of the includes if none were specified. Kill it now. Kill it with fire!

Reply:
💥